packages/schematics/angular/library/files/__projectRoot__/README.md

```diff
@@ -0,0 +1,24 @@
+# <%= dasherize(name) %>
```

Comment:
I think for consistence this should be `classify`, similar to the workspace readme.

Reply:
Changed!